src/devtools/client/webconsole/components/Output/ConsoleOutput.tsx

```diff
@@ -151,36 +187,29 @@ class ConsoleOutput extends React.Component<PropsFromRedux> {
       loadedRegions,
     } = this.props;
 
-    const messageNodes = visibleMessages
-      .filter(messageId => {
-        const time = messages.entities[messageId]!.executionPointTime!;
-        return loadedRegions!.loaded.some(
-          region => time >= region.begin.time && time <= region.end.time
-        );
-      })
-      .map((messageId, i) => {
-        const message = messages.entities[messageId]!;
-        // @ts-ignore ExecutionPoint/string mismatch
-        const isPrimaryHighlighted = hoveredItem?.point === message.executionPoint;
-        const shouldScrollIntoView = isPrimaryHighlighted && hoveredItem?.target !== "console";
-
-        return React.createElement(MessageContainer, {
-          dispatch,
-          key: messageId,
-          messageId,
-          message,
-          open: messagesUi.includes(messageId),
-          // TODO Reconsider this when we rebuild message grouping
-          payload: undefined,
-          timestampsVisible,
-
-          pausedExecutionPoint,
-          isPaused: closestMessage?.id == messageId,
-          isFirstMessageForPoint: this.getIsFirstMessageForPoint(i, visibleMessages),
-          isPrimaryHighlighted,
-          shouldScrollIntoView,
-        });
+    const messageNodes = visibleMessages.map((messageId, i) => {
+      const message = messages.entities[messageId]!;
+      // @ts-ignore ExecutionPoint/string mismatch
+      const isPrimaryHighlighted = hoveredItem?.point === message.executionPoint;
+      const shouldScrollIntoView = isPrimaryHighlighted && hoveredItem?.target !== "console";
```

Comment:
Looks like we store "hovered item" in the Redux state (`state.timeline.hoveredItem`). FWIW I didn't write this, just moved the surrounding code a bit.
https://github.com/RecordReplay/devtools/blob/1b65b594016980128185df9b7baaf093e5d04e36/src/devtools/client/webconsole/components/Output/ConsoleOutput.tsx#L154-L183

My initial guess is that this check is to prevent the logs from scrolling/jumping while the user is hovering over the console (perhaps inspecting or selecting some logged text) but as I log that value, it always appears to be `undefined` when I'm I test things in the console so 🤷🏼 

cc @hbenl (45ddaa2ad6ab521ad4fdd347b746c9fd65dc8adf) for a better explanation maybe?